behat.yml.dist

```diff
@@ -6,6 +6,7 @@ default:
       contexts:
         - Drupal\DrupalExtension\Context\MinkContext
         - Drupal\DrupalExtension\Context\DrupalContext
+        - Drupal\Tests\oe_authentication\Behat\OeAuthenticationContext
```

Comment:
Convention for Drupal modules is `namespace Drupal\Tests\oe_authentication\...`, I'd also remove the `Oe` in `OeAuthenticationContext` since it's already in the namespace.